src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -651,8 +651,11 @@ def __init__(self,
       self.size = size
     else:
       gcs_object = get(get_cloud_storage_file_path(bucket, object_path))
-
-      self.filename = gcs_object['metadata'].get(BLOB_FILENAME_METADATA_KEY)
+      if 'metadata' in gcs_object.keys():
+        self.filename = gcs_object['metadata'].get(BLOB_FILENAME_METADATA_KEY,
+                                                   None)
+      if not self.filename:
```

Comment:
I don't think we should log if it's totally normal behavior.

Reply:
Could you confirm if filename is ever necessary? I didn't find a reason to believe so. 
I can remove it now, or we can leave it and then remove it later, as you wish.

Reply:
Let's remove later and land now.